src/common/components/orbit.h

```diff
@@ -4,37 +4,53 @@
 #pragma once
 
 #include <entt/entt.hpp>
-
+#include <math.h>
 #include "common/components/units.h"
+#include "spdlog/spdlog.h"
```

Comment:
Unneeded include, also don't delete spaces unless needed.

Reply:
Needed for period calculations later in the code.